js/UX/union.js

```diff
@@ -100,5 +100,13 @@ function intersectar(elemento1, elemento2){
 			resultado.push(elemento2[i]);	
 		}
 	}
-	$("#resultado").html("{"+elemento1+"} ∩"+" {"+elemento2+"} = {"+resultado+"}");//se imprime el resultado en a card
+	imprimir(["{",elemento1,"}"," ∩ ","{",elemento2,"}"," = ","{",resultado,"}"],11);
+}
+
+function imprimir(vector,tam){
+	resultado="";
+	for(i=0;i<tam;i++){
+		resultado= resultado + vector[i];
+	}
+	$("#resultado").html(resultado);
```

Comment:
quita el parametro " tam " y en vez de usar for, usa un foreach y te ahorras meter una variable con el tamaño